jdi-eyes-demo/src/main/java/jdisite/entities/MarvelUserInfo.java

```diff
@@ -0,0 +1,7 @@
+package jdisite.entities;
+
+import com.epam.jdi.tools.DataClass;
+
+public class MarvelUserInfo extends DataClass<MarvelUserInfo> {
+    public String number, type, user, description;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)